.github/configs/README.md

```diff
@@ -0,0 +1,16 @@
+# Build Configuration Files
+
+This directory contains configuration files used by the CI/CD pipeline for build validation and testing.
+
+-   Defines shader compilation parameters, known warnings, and expected build configurations
+-   Used by the GitHub Actions workflow to validate shader compilation during CI builds
+-   Not intended for direct user interaction - this is build infrastructure configuration
+
+## Files
+
+-   `shader-validation.yaml`: Configuration for shader compilation validation using hlslkit
+-   `shader-validation-vr.yaml`: VR Configuration for shader compilation validation using hlslkit
+
+## Usage
```

Comment:
Can you include the command with the same args you used to generate `shader-validation.yaml` and `shader-validation-vr.yaml`? Or even include a bat file that does this to make this step easier when updating shaders/defines?